main.c

```diff
@@ -195,11 +195,47 @@ void add_log(struct board_tile board[SIZE][SIZE], int turtle_row[SIZE]) {
     }
 }
 
-void clear_row(struct board_tile board[SIZE][SIZE]) {
-    printf("// TODO clear_row [row]\n");
+void clear_row(
+    struct board_tile board[SIZE][SIZE],
+    int last_coordinate[2]
+) {
+    int row;
+    printf("Enter row to clear: ");
+    scanf("%d", &row);
+
+    // Check if row is out of bounds
+    if (row < 0 || row >= SIZE) {
+        printf("Error: Row %d is out of bounds.\n", row);
+        return;
+    }
+
+    // Check if row is prohibited (LILLYPAD or BANK row)
+    if (row == LILLYPAD_ROW || row == BANK_ROW) {
+        printf("Error: Cannot clear the Lilypad or Bank row.\n");
+        return;
+    }
```

Comment:
since we already have a helper that will let us know if we can do certain operation on certain row or not, we can just leverage the `is_placeable` function:

the `is_placeable` function accepts row or column, the row is from the input, while the column you can use any column, so choose any

```suggestion
    if (is_placeable(row, LEFT_COLUMN) == FALSE) {
        printf("Error: Cannot clear the Lilypad or Bank row.\n");
        return;
    }
```